### PR TITLE
gweled: unstable-2018-02-15 -> unstable-2021-02-11

### PR DESCRIPTION
**Pull request by SFrijters**
###### Motivation for this change

ZHF: #122042

Updated to latest version, which introduces a dependency on gettext, with a hardcoded version in the autoconf file.
After this update and fix, the game builds and I can run the executable, but I'm seeing graphics corruption in the game itself.

Currently there are no maintainers and I don't particularly feel like becoming the new one and debugging this issue, so if others are also seeing these graphics issues maybe the package should be dropped instead.

@NixOS/nixos-release-managers 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/games/gweled/default.nix [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
   doCheck = false;
 
+  postPatch = ''
+    substituteInPlace configure.ac --replace "AM_GNU_GETTEXT_VERSION([0.19.8])" "AM_GNU_GETTEXT_VERSION([${gettext.version}])"
+  '';
+
   nativeBuildInputs = [ wrapGAppsHook intltool autoreconfHook pkg-config ];
 
-  buildInputs = [ gtk2 libmikmod librsvg hicolor-icon-theme libcanberra-gtk2 ];
+  buildInputs = [ gtk2 libmikmod librsvg hicolor-icon-theme libcanberra-gtk2 gettext ];
```
Comment (jonringer) [inline pkgs/games/gweled/default.nix:23]: gettext should go in nativeBuildInputs. Usually just needs to be ran as part of a build.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 122114` at ef0a4b32 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>gweled</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/games/gweled/default.nix:32:5:
```
   |
32 |     maintainers = [ ];
   |     ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl2` is a deprecated license, please check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Near pkgs/games/gweled/default.nix:30:5:
```
   |
30 |     license = licenses.gpl2;
   |     ^
```
</li>
  </ul>
</details>

File: pkgs/games/gweled/default.nix (hunk 1 above)
Comment (SFrijters) [reply pkgs/games/gweled/default.nix]: Oops, fixed. Also removed `intltool` since `gettext` replaced it.

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 122114 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>1 package built:</summary>  <ul>    <li>gweled</li>  </ul></details>

**Pull request merged**